Configuration/Geometry/python/GeometrySimDB_cff.py

```diff
@@ -7,3 +7,5 @@
 from GeometryReaders.XMLIdealGeometryESSource.cmsGeometryDB_cff import *
 from Geometry.MuonNumbering.muonNumberingInitialization_cfi import *
 from Geometry.TrackerNumberingBuilder.trackerNumberingGeometryDB_cfi import *
+from Geometry.HcalCommonData.hcalParameters_cfi      import *
```

Comment:
I have updated this PR taking care of Yana's comments. Can this PR be tested and integrated soon?
________________________________
From: Ianna Osborne [notifications@github.com]
Sent: 20 July 2015 10:11
To: cms-sw/cmssw
Cc: Sunanda Banerjee
Subject: Re: [cmssw] bsunanda:Run2-hcs23 Modify to add the adequate ESProducers in the configuration scenarios (#10267)


In Configuration/Geometry/python/GeometrySimDB_cff.py<https://github.com/cms-sw/cmssw/pull/10267#discussion_r34974461>:

> @@ -7,3 +7,5 @@
>  from GeometryReaders.XMLIdealGeometryESSource.cmsGeometryDB_cff import *
>  from Geometry.MuonNumbering.muonNumberingInitialization_cfi import *
>  from Geometry.TrackerNumberingBuilder.trackerNumberingGeometryDB_cfi import *
> +from Geometry.HcalCommonData.hcalParameters_cfi      import *


@bsunanda<https://github.com/bsunanda> - the same here, HcalParameters will be picked up from DB

—
Reply to this email directly or view it on GitHub<https://github.com/cms-sw/cmssw/pull/10267/files#r34974461>.